element/src/commonMain/kotlin/RootElement.kt

```diff
@@ -0,0 +1,16 @@
+package com.juul.krayon.element
+
+import com.juul.krayon.kanvas.Kanvas
+
+public class RootElement : Element() {
+
+    override val tag: String get() = "root"
+
+    override fun <PAINT, PATH> draw(canvas: Kanvas<PAINT, PATH>) {
+        descendents.forEach { it.draw(canvas) }
```

Comment:
How does this resolve in terms of z-height? If each `Element` only has a single descendant, it's pretty straight forward, but what about when  multiple `Elements` have many descendants?

Also, with a lot of overlapping `Elements`, this seems like it could have some performance implications.

Reply:
Z-order is implicit and not something we can control. I got lucky, and the easiest-to-implement behavior matched the SVG spec:

> Elements in an SVG document fragment have an implicit drawing order, with the first elements in the SVG document fragment getting "painted" first. Subsequent elements are painted on top of previously painted elements.

Source: https://www.w3.org/TR/SVG11/render.html#RenderingOrder